setup.py

```diff
@@ -24,7 +24,7 @@
 install_requires = [
     'diff-match-patch',
     'Django>=2.0',
-    'tablib<1.0.0',
+    'tablib>=0.14.0',
```

Comment:
@claudep shouldn't we do tablib[all] like in `requirements/base.txt`? a lot of people are having problems because `xlrd` doesn't get installed automatically...

Reply:
@claudep you are right. @andrewgy8 @bmihelac what do you think?

This is ready to merge, probably for a 2.1 release, but users would still need to specify extra dependencies manually, which is not happening now with the 2.0.1 release because we are preventing tablib 1.0.0 from being installed.